rollbar/__init__.py

```diff
@@ -287,7 +287,8 @@ def _get_pylons_request():
     'locals': {
         'enabled': True,
         'safe_repr': True,
-        'sizes': DEFAULT_LOCALS_SIZES
+        'sizes': DEFAULT_LOCALS_SIZES,
+        'whitelisted_types': []
```

Comment:
'whitelisted_types' is what I want it to be however SerializableTransform uses 'whitelist_types'. I didn't want to cause a breaking change so I kept the later but used the new name for a new config option.